src/erlang_ls_code_navigation.erl

```diff
@@ -10,6 +10,11 @@
 %% API
 -export([ goto_definition/2 ]).
 
+-export([ app_path/0
```

Comment:
These are exported so that the indexer process can acces them. The indexer currently only uses `app_path/0` though.